client/ayon_core/plugins/publish/collect_resources_path.py

```diff
@@ -64,7 +64,8 @@ class CollectResourcesPath(pyblish.api.InstancePlugin):
                 "skeletalMesh",
                 "xgen",
                 "yeticacheUE",
-                "tycache"
+                "tycache",
+                "OxRig"
```

Comment:
No other families seem to start with a a capital letter? Should this also start with lowercase?